scalekit/v1/workspaces/workspaces_pb2_grpc.py

```
@@ -135,13 +145,44 @@ def GetBillingPortal(self, request, context):
+        Behavior notes:
+        - All amounts are returned in minor currency units with accompanying currency code.
+        - current_invoice_preview is not a finalized invoice and may change until the period closes.
+        - payment_methods indicate availability and default selection; use is_default to preselect for checkout or updates.
+        - Use billing_period dates to compute prorated changes when updating plans mid-cycle.
+
+        """
         context.set_code(grpc.StatusCode.UNIMPLEMENTED)
         context.set_details('Method not implemented!')
         raise NotImplementedError('Method not implemented!')
 
     def GetProductUsage(self, request, context):
```

Ruff (0.14.8)

Unused method argument: `request`

(ARG002)